### DISCUSSION TITLE
Help required

### DISCUSSION THREAD

**Discussion by Emeronerp1** [serenity-is/Serenity#6665] [Help] [answered]
![IMG-20230123-WA0035.jpg](https://user-images.githubusercontent.com/45841416/213975194-95a02bb3-e640-4756-baf9-9116d362d001.jpg)

I created an endpoint with the Route as shown in Image, when i comolie the route in service script is showing HasNoRoute/SalesHelperController.

Actually I specified the Route as
Route("Services/Sales/SalesHelper/[action]")

When I call this service in another form dialog causing error because I can see the service is called as
services/HasNoRoute/SalesHelperList

I didn't get what is the issue, did I miss something.

I given read permission in the row as well

**Comment by VictorTomaili** [comment, marked as answer]
remove `using Microsoft.AspNetCore.Components;` line and rebuild.

**Reply by Emeronerp1** [reply]
It worked.
Thank you
